src/commands/countdown/subcommands/list.ts

```diff
@@ -0,0 +1,62 @@
+import dayjs from 'dayjs';
+import * as Discord from '../../../Discord';
+import SubCommand from '../../../SubCommand';
+import { bold, fancyCenter } from '../../../format';
+
+type MonthGroup = {
+  month: string;
+  year: number;
+  heading: string;
+  list: string;
+};
+
+const list = new SubCommand({
+  name: 'list',
+  displayName: 'List Countdowns',
+  description: 'List all stored countdowns',
```

Comment:
`stored` or `current`? Should completed countdowns be automatically deleted or remain until removed by a mod?

Reply:
Automatically deleted, and "current" ! Good find, making the changes